crypto/fipsmodule/hmac/hmac.c

```diff
@@ -319,6 +319,8 @@ int HMAC_Init_ex(HMAC_CTX *ctx, const void *key, size_t key_len,
 
   result = 1;
 end:
+  OPENSSL_cleanse(pad, EVP_MAX_MD_BLOCK_SIZE / 8);
```

Comment:
Could probably soon argue that defining e.g. `EVP_MAX_MD_BLOCK_SIZE_BYTES` would be useful instead of `X / 8` all the time.

Or an `ARRAY_SIZE()` macro.

Reply:
Sure, I had been briefly contemplating creating a `EVP_MAX_MD_BLOCK_SIZE_BYTES` before making this change. I'll add it now.